### ISSUE TITLE
Look into license reporting for deps

### ISSUE THREAD

**Issue by lesserwhirls** [Unidata/thredds#647]
We have been asked to collect license info for our dependencies. There is a gradle plugin that will generate reports for us called "license-gradle-plugin":

https://github.com/hierynomus/license-gradle-plugin

It does much more that dep license inspection and reporting, but that's all we would need for now. @cwardgar - would you be interested in taking a stab?

**Comment by lesserwhirls** [comment]
I could get this plugin to work, but I don't think I'm doing gradle the way the rest of the project is doing gradle.  See [here](https://github.com/lesserwhirls/thredds/commit/241e08a2e6c3a39cd6e67bf338aa29e168aa1be1) for my attempt. I should really learn more about gradle.

**Comment by cwardgar** [comment]
#681 fixes our outstanding uses of GPLed code. Is there more to do here? Perhaps add some aliases to make the report look nice and add it to our docs?

**Comment by lesserwhirls** [comment]
Yep. The aliases will be helpful in reducing the clutter. The html file can be added (I think) directly to the jekyll docs and "beautified" there.
